types/telegram-web-app/index.d.ts

```diff
@@ -635,103 +635,130 @@ export type FileDownloadRequestedCallback = (eventData: {
      */
     link_color?: string;
+
     /**
      * Button color in the `#RRGGBB` format.
-     * Also available as the CSS variable `var(--tg-theme-button-color)`.
+     *
+     * @see Available as the CSS variable `var(--tg-theme-button-color)`.
      */
     button_color?: string;
+
     /**
      * Button text color in the `#RRGGBB` format.
-     * Also available as the CSS variable `var(--tg-theme-button-text-color)`.
+     *
+     * @see Available as the CSS variable `var(--tg-theme-button-text-color)`.
      */
     button_text_color?: string;
+
     /**
-     * **Bot API 6.1+**
-     *
      * Secondary background color in the `#RRGGBB` format.
-     * Also available as the CSS variable `var(--tg-theme-secondary-bg-color)`.
+     *
+     * @since Bot API 6.1+
+     *
+     * @see Available as the CSS variable `var(--tg-theme-secondary-bg-color)`.
      */
     secondary_bg_color?: string;
+
     /**
-     * **Bot API 7.0+**
-     *
      * Header background color in the `#RRGGBB` format.
-     * Also available as the CSS variable `var(--tg-theme-header-bg-color)`.
+     *
+     * @since Bot API 7.0+
+     *
+     * @see Available as the CSS variable `var(--tg-theme-header-bg-color)`.
      */
     header_bg_color?: string;
+
     /**
-     * **Bot API 7.10+**
+     * Bottom background color in the `#RRGGBB` format.
+     *
+     * @since Bot API 7.10+
      *
-     * Bottom background color in the #RRGGBB format.
-     * Also available as the CSS variable var(--tg-theme-bottom-bar-bg-color).
+     * @see Available as the CSS variable `var(--tg-theme-bottom-bar-bg-color)`.
      */
     bottom_bar_bg_color?: string;
+
     /**
-     * **Bot API 7.0+**
-     *
      * Accent text color in the `#RRGGBB` format.
-     * Also available as the CSS variable `var(--tg-theme-accent-text-color)`.
+     *
+     * @since Bot API 7.0+
+     *
+     * @see Available as the CSS variable `var(--tg-theme-accent-text-color)`.
      */
     accent_text_color?: string;
+
     /**
-     * **Bot API 7.0+**
+     * Section background color in the `#RRGGBB` format.
+     *
+     * @since Bot API 7.0+
      *
-     * Background color for the section in the `#RRGGBB` format.
-     * It is recommended to use this in conjunction with *secondary_bg_color*.
-     * Also available as the CSS variable `var(--tg-theme-section-bg-color)`.
+     * @see Available as the CSS variable `var(--tg-theme-section-bg-color)`.
      */
     section_bg_color?: string;
+
     /**
-     * **Bot API 7.0+**
+     * Section header text color in the `#RRGGBB` format.
+     *
+     * @since Bot API 7.0+
      *
-     * Header text color for the section in the `#RRGGBB` format.
-     * Also available as the CSS variable `var(--tg-theme-section-header-text-color)`.
+     * @see Available as the CSS variable `var(--tg-theme-section-header-text-color)`.
      */
-    section_header_text_color?: `#${string}`;
+    section_header_text_color?: string;
```

Comment:
What was wrong with the more specific type?

Reply:
for uniformity with other fields
___
there is no input validation
U can set any string value 
```js
Telegram.WebApp.themeParams.bg_color = 'red'
> 'red'
Telegram.WebApp.themeParams.bg_color
> 'red'
```
idk for what 🤷

Reply:
But does Telegram support `"red"`?